OMEdit/OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp

```diff
@@ -182,49 +192,43 @@ void GraphicsView::setIsCreatingTransition(bool enable)
 void GraphicsView::setIsCreatingLineShape(bool enable)
 {
   mIsCreatingLineShape = enable;
-  setDragModeInternal(enable);
-  setItemsFlags(!enable);
-  updateUndoRedoActions(enable);
+  setIsCreatingPrologue(enable);
```

Comment:
Call this function also in `setIsCreatingConnection()` and `setIsCreatingTransition()`.

Reply:
Done